CONTRIBUTING.rst

```diff
@@ -68,8 +68,18 @@ We recommend the following workflow:
    c. Document any user-facing changes in one of the ``/docs/`` files.
       Please use `one sentence per line`_.
 
-   d. Create a newsfragment in ``src/towncrier/newsfragments/`` describing the changes and containing information that is of interest to end-users.
+   d. Create a news fragment in ``src/towncrier/newsfragments/`` describing the changes and containing information that is of interest to end-users.
       Please use `one sentence per line`_ here, too.
```

Comment:
I don't remember which company was describing/documenting the style for writing documentation. But I remember, that the suggestion was to avoid "please" in the docs 

```suggestion
      Use `one sentence per line`_ here, too.
```